llm/python/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do we need to use MIT license?

Reply:
genai repo licensed under Apache2.0 https://github.com/openvinotoolkit/openvino.genai/blob/master/LICENSE
is it not enough for publishing llm bench code?

Reply:
LICENSE has been removed